src/filter_tree/filter_tree.c

```diff
@@ -26,7 +26,7 @@ FT_FilterNode* CreateCondFilterNode(int op) {
     return filterNode;
 }
 
-FT_FilterNode* _CreatePredicateFilterNode(const AST_PredicateNode *pn, const QueryGraph *qg) {
+FT_FilterNode* _CreatePredicateFilterNode(const AST_PredicateNode *pn) {
```

Comment:
Nice catch, I didn't realize this argument wasn't doing anything! We should delete the include in the header file too (line 13).

Reply:
👍